### DISCUSSION TITLE
Discord RPC shows wrong icon?

### DISCUSSION THREAD

**Discussion by shertowns** [ppy/osu#28526] [Q&A] [answered]
![изображение](https://github.com/ppy/osu/assets/86488248/01a40881-e1c6-4b22-8669-6f644e67895e)
iirc the dot got removed

**Comment by SupDos** [comment]
They are [aware of it](https://discord.com/channels/188630481301012481/1097318920991559880/1250813341538975895)

**Reply by peppy** [reply, marked as answer]
Should be updated now.
